x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list_drawer/most_recent_error.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { MonitorPageLink } from '../monitor_page_link';
+import { useUrlParams } from '../../../../hooks';
+import { stringifyUrlParams } from '../../../../lib/helper/stringify_url_params';
+
+interface RecentError {
+  message: string;
+  type: string;
```

Comment:
I mention in my overall review comment that it would be really nice to know _when_ this happened, so a timestamp prop here would be awesome. That can be a low-fruit enhancement IMO as long as we catalogue it in an issue targeted at the next release.